### DISCUSSION TITLE
Installing Nominatim: How to Keep Only the Necessary Data

### DISCUSSION THREAD

**Discussion by MirroxDev** [osm-search/Nominatim#3561] [General]
Is it possible, when installing Nominatim on a VPS, to limit the download to only the data we need, such as:

Countries

Regions

Settlements

Here’s an example of the data we need:

Country code

Region code

Settlement name

Alternative names

Longitude

Latitude

Postal code

Can we exclude all detailed information, like restaurants, roads, factories, and avoid downloading these data?

**Comment by mtmail** [comment]
Have a look at the "Filtering Imported Data" section on https://nominatim.org/release-docs/latest/admin/Import/ I think you need the 'admin' style. It will include neighborhoods/suburbs, too. Best to test the import for one country first before importing the whole world (planet).
